src/utils/unit-converter.tsx

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
No need to do a tsx file, tsx is for react gestion, .ts should work

Reply:
Indeed, renamed.